relayer/scripts/devnet-relayers-compose.yml

```diff
@@ -7,21 +7,21 @@ services:
       - DEV_MODE=true
     image: most-relayer
     network_mode: host
-    restart: on-failure
+    restart: on-failure:3
```

Comment:
the effect of the previous flag would be that in case of a broken relayer it would be stuck in a constant restart loop, which is not immediately evident when listing docker processes and obfusates docker logs, that might aid in triangulating what went wrong.  This way the process is gone after trying 3 times and you can do a post-mortem easily.